docker_custodian/docker_gc.py

```diff
@@ -77,6 +77,8 @@ def get_all_images(client):
 def get_dangling_volumes(client):
     log.info("Getting dangling volumes")
     volumes = client.volumes({'dangling': True})['Volumes']
+    if volumes == None:
+        volumes = []
```

Comment:
You could also do this on the line above:

```python
volumes = client.volumes({'dangling': True})['Volumes'] or []
```

Reply:
Thanks.  I'm a Python noob and knew there must be a better way.
I figured that out yesterday but hadn't pushed it yet.